<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedenDialog</class>
 <widget class="QDialog" name="LedenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>611</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbToevoegen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Toevoegen</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRec">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWijzig">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Wijzig</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbVerwijderen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTest">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
